src/SOS/Strike/util.h

```diff
@@ -298,14 +299,217 @@ class CachedString
     // -1 - mPtr points to a pointer we have new'ed
     // -2 - We hit an oom trying to allocate either mCount or mPtr
     int mIndex;
-    
+
     // contains the size of current string
     int mSize;
 
 private:
     static StaticData<char, 4, 1024> cache;
 };
 
+class GCHeapDetails
+{
+private:
+    void GetGenerationTableSize(CLRDATA_ADDRESS svrHeapAddr, unsigned int *count)
```

Comment:
>GetGenerationTableSize [](start = 9, length = 22)

nit - what was the reason to call this `GetGenerationTableSize` instead of `GetNumberGenerations` 'cause it seems like that's exactly what it does?

Reply:
It's a helper method in GCHeapDetails and not a public API, and the only place it's called is to get the size to allocate for the generation table. It could be named either way